api/envoy/extensions/filters/http/jwt_authn/v3/config.proto

```diff
@@ -528,4 +529,25 @@ message JwtAuthentication {
   // <http://www.w3.org/TR/cors/#cross-origin-request-with-preflight>`_ regardless of JWT
   // requirements specified in the rules.
   bool bypass_cors_preflight = 4;
+
+  // A map of unique requirement_names to JwtRequirements.
+  // :ref:`requirement_name <envoy_v3_api_field_extensions.filters.http.jwt_authn.v3.PerRouteConfig.requirement_name>`
+  // in `PerRouteConfig` uses this map to specify a JwtRequirement.
+  map<string, JwtRequirement> requirement_map = 5;
+}
+
+// Specify per-route config.
+message PerRouteConfig {
+  oneof requirement_specifier {
+    option (validate.required) = true;
+
+    // Bypass Jwt Authentication for this route.
+    bool bypass = 1;
```

Comment:
If this is set this must be `true`? Add a PGV rule for that. Name this `disabled`, so it is more consistent with ext_authz.

Reply:
Done